types/logger.go

```diff
@@ -29,12 +44,21 @@ func NewKairosLogger(name, level string, quiet bool) KairosLogger {
 		logName := fmt.Sprintf("%s.log", name)
 		_ = os.MkdirAll("/var/log/kairos/", os.ModeDir|os.ModePerm)
 		logFileName := filepath.Join("/var/log/kairos/", logName)
-
 		logfile, err := os.OpenFile(logFileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 		if err == nil {
 			loggers = append(loggers, zerolog.ConsoleWriter{Out: logfile, TimeFormat: time.RFC3339, NoColor: true, FieldsExclude: []string{"SYSLOG_IDENTIFIER"}})
 		}
+	}
 
+	// Add extra dirs to the logger
+	for _, dir := range dirs {
```

Comment:
Why is `/var/log/kairos` completely skipped when we have journal but these are not?
If we decide to treat `/var/log/kairos` the same as the rest of the `dirs`, you can just add it to the list and avoid the duplication between the `else` block above and the block here. In other words, merge all dirs in one slice including `/var/log/kairos`.

Reply:
Basically to not duplicate logs, we either log to the journal or to a file. So this works on systemd systems, on openrc systems, inside a docker container, on UKI and so on.

Basically we cover everything possible for Kairos workflows, plus we allow extending for specific workflows that may need extra logging places, like immucore in uki